Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragmentViewModel.kt

```diff
@@ -321,6 +339,13 @@ class HomeFragmentViewModel @AssistedInject constructor(
         cwaSettings.wasTracingExplanationDialogShown = true
     }
 
+    fun isOlderThanThreshold(
+        hours: Duration,
+        testTimestamp: Instant,
+        timeNow: Instant = Instant.now()
+    ): Boolean =
+        Days.daysBetween(testTimestamp, timeNow).toStandardDuration() >= hours
```

Comment:
calculation should be with hours not days, if you add unit test with hours not days it will fail